docs/platform-hyperkit.md

```diff
@@ -54,7 +80,11 @@ docker run --rm -ti -v ~/.ssh:/root/.ssh  ssh ssh <IP address of VM>
 While VPNKit has the general tooling to expose any VMs port on the
 localhost (just like it does with containers in Docker for Mac), we
 are unlikely to expose this as a general feature in `linuxkit run` as
-it is very specific to the macOS. However, you can use a `socat` container to proxy between LinuxKit VMs ports and localhost.  For example, to expose the redis port from the [RedisOS example](../examples/redis-os.yml), use this Dockerfile:
+it is very specific to the macOS. However, you can use a `socat`
```

Comment:
I #2008 ready to be merged? If so I handle the merge. I do would like to have the hyperkit (and other docs) follow a common pattern